vision_agent/agent/vision_agent_coder_prompts.py

```diff
@@ -323,17 +342,24 @@ def find_text(image_path: str, text: str) -> str:
 ```
 
 It raises this error:
+```
 {result}
+```
 
 This is previous feedback provided on the code:
 {feedback}
 
-Please fix the bug by follow the error information and return a JSON object with the following format:
+Please fix the bug by correcting the error. Return the following JSON object followed by the fixed code in the below format:
+```json
 {{
     "reflections": str # any thoughts you have about the bug and how you fixed it
-    "code": str # the fixed code if any, else an empty string
-    "test": str # the fixed test code if any, else an empty string
+    "which_code": str # which code you fixed, can either be 'code' or 'test'
```

Comment:
OK i can see what "which_code" means. I would rephrase this as 
`"which_code": str # The code block that was fixed, can only be 'code' or 'test'`